File: ScaleformUI_Lua/src/Menus/PauseMenus/PauseMenu/Items/SettingsItems/SettingsSeparatorItem.lua
SettingsSeparatorItem = {}
SettingsSeparatorItem.__index = SettingsSeparatorItem
setmetatable(SettingsSeparatorItem, { __index = SettingsItem })
SettingsSeparatorItem.__call = function() return "SettingsSeparatorItem" end

function SettingsSeparatorItem.New(label)
    local base = SettingsItem.New(label, "")
    if label:IsNullOrEmpty() then
        base.ItemType = SettingsItemType.Empty
    else
        base.ItemType = SettingsItemType.Separator
    end
    base.IsJumpable = false
    return setmetatable(base, SettingsSeparatorItem)
end

File: ScaleformUI_Csharp/Menus/Pause Menus/PauseMenu/Items/SettingsItems/SettingsSeparatorItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScaleformUI.PauseMenu
{
    public class SettingsSeparatorItem : SettingsItem
    {
        public bool IsJumpable { get; private set; }
        public SettingsSeparatorItem(string label) : base(label, "")
        {
            ItemType = SettingsItemType.Separator;
            Label = label;
        }
        public SettingsSeparatorItem() : base("", "")
        {
            ItemType = SettingsItemType.Empty;
        }
    }
}